### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by OpenSauce**
Re-wording

I'm trying it as succinct, powerful, and relatable as possible

### CODE REVIEW

**Comment by OpenSauce** [comment]
Do we have some project based images? Like a banner, logo etc?

**Review by ptrfarkas — APPROVED** [review]
Awesome, looks good!

File: README.md [hunk 1]
```diff
@@ -1,38 +1,38 @@
...
+MangoDB is in its very early stages and welcomes all contributors.
+See our [CONTRIBUTING.md](CONTRIBUTING.md).
 
 
 ## Quickstart
 
-Those steps describe a quick local setup.
-They are not suitable for production use.
+These steps describe a quick local setup.
+They are not yet suitable for production use.
```
[the remark this answers is not in the document]
Comment (OpenSauce) [reply README.md:35]: But is the aim to one day be suitable for production use?

[the remark this answers is not in the document]
Comment (OpenSauce) [reply README.md:35]: Not yet just implies they are not suitable... For now :smile:

[the remark this answers is not in the document]
Comment (OpenSauce) [reply README.md:35]: @mangodb-bot I understand what you're saying, one day there will be a solution for production use.

[the remark this answers is not in the document]
Comment (OpenSauce) [reply README.md:35]: Or even better yet - let's remove talking about production. We can describe a quick local setup - anything else we can build upon :smiley: How's that sound?

[the remark this answers is not in the document]
Comment (OpenSauce) [reply README.md:35]: Your call ;)

[the remark this answers is not in the document]
Comment (OpenSauce) [reply README.md]: My pleasure!

**Pull request merged**